### PR TITLE
[BPK-2830]: Removed gray theming from all components

### PR DESCRIPTION
**Pull request by ojcurtis**
Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### CODE REVIEW

File: packages/bpk-component-map/src/BpkMapMarker.scss [hunk 1]
```diff
@@ -55,18 +55,14 @@
   }
 
   &--plain {
-    background-color: $bpk-color-gray-700; // For IE
```
Comment (georgegillams) [inline packages/bpk-component-map/src/BpkMapMarker.scss]: Surely IE still needs this?

Comment (ojcurtis) [reply packages/bpk-component-map/src/BpkMapMarker.scss]: Nope as this was added due to the lack of support for `var()` but now it has been removed it is no longer needed.

See this PR https://github.com/Skyscanner/backpack/pull/1463

Comment (ojcurtis) [reply packages/bpk-component-map/src/BpkMapMarker.scss]: Nope as this was added due to the lack of support for `var()` so we had to add this in order for the fallback.

See this PR wheere it was added https://github.com/Skyscanner/backpack/pull/1463

**Comment by ojcurtis** [comment]
For reference it is reverting the work that was completed in this PR: https://github.com/Skyscanner/backpack/pull/1463

**Pull request merged**